stage0/run.go

```diff
@@ -47,21 +43,17 @@ import (
 	"github.com/coreos/rocket/pkg/lock"
 	ptar "github.com/coreos/rocket/pkg/tar"
 	"github.com/coreos/rocket/version"
-
-	"github.com/coreos/rocket/stage0/stage1_init"
-	"github.com/coreos/rocket/stage0/stage1_rootfs"
 )
 
 const (
-	initPath  = "stage1/init"
+	initPath  = "stage1/rootfs/init"
```

Comment:
Yes, it's awkward, so I don't discover the entrypoints from the manifest (yet).

Maybe we should just define some annotations where an image can specify rocket stage1 entrypoints by name, which would currently be just `rocketInit` and `rocketEnter`?  Then any .aci which fulfills those annotations can be used as a rocket stage1?